README.md

```diff
@@ -6,3 +6,9 @@ Sample Jupyter Notebooks to inspect data of your Home Assistant instance.
 
 - For help to get started, see the [Home Assistant Data Science Portal](https://data.home-assistant.io)
 - Package to help you explore Home Assistant data: [Hass Data Detective](https://github.com/robmarkcole/HASS-data-detective)
+
+## Development
+* Create a venv: `python3 -m venv venv`
+* Activate venv: `source venv/bin/activate`
+* Install Jupyter to run the notebooks: `pip3 install jupyterlab`
+* Run jupyter, ensuring from venv: `venv/bin/jupyter lab`
```

Comment:
This is already present at https://data.home-assistant.io/docs/quick-start-core#preparing-the-user-environment.

I'm not in favor for adding this as it would mean that we have to maintain it at two places.

Reply:
removed